ui/store/actions.ts

```diff
@@ -2083,7 +2083,7 @@ export function addNftVerifyOwnership(
 export function removeAndIgnoreNft(
   address: string,
   tokenID: string,
-  dontShowLoadingIndicator: boolean,
+  dontShowLoadingIndicator?: boolean,
```

Comment:
Optional: could rename it as 
```suggestion
  shouldShowLoadingIndicator?: boolean,
```